PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -242,7 +242,7 @@ DECLARE_SOA_DYNAMIC_COLUMN(MIDBoardCh3, midBoardCh3, //!
                            [](uint32_t midBoards) -> int { return static_cast<int>((midBoards >> 16) & 0xFF); });
 DECLARE_SOA_DYNAMIC_COLUMN(MIDBoardCh4, midBoardCh4, //!
                            [](uint32_t midBoards) -> int { return static_cast<int>((midBoards >> 24) & 0xFF); });
-
+DECLARE_SOA_SELF_INDEX_COLUMN_FULL(MCHTrack, matchMCHTrack, int, "Muons_MatchMCHTrack");
```

Comment:
I don' think this nomenclature will work with the aod merger which means that you cannot merge derived data (and therefore not produce it on hyperloop). This is because here you call it `Muons` and below it is called actually `RTMUON`.

Can you test this by producing two derived AODs (from two different input files) and the merge it using `o2-aod-merger`?

Can you test this by merging two

Reply:
@iarsene @batoul-d

Reply:
@jgrosseo  From my discussion with @batoul-d, she tested this, but can you confirm?

Reply:
A merging test is now also included in Hyperloop but this issue is not flagged as error, you should see a warning in the output though. If you made a test, you can send me the stdout.log and I can check (you find this at the end).

Reply:
the name of this branch is indeed a problem for the merging. This pull request does not try to fix this issue but focuses on filling the index correctly. The compatibility of indices' names in the reduced files concerns all the indices (self- or not) so the next step is to fix that. From what I saw already, the change needs to be done on both sides, the branches and the tool itself. For example the tool explicitly adds the label"O2" when it tries to find the tree to which the index is pointing to (https://github.com/AliceO2Group/O2Physics/blob/master/Common/Core/aodMerger.cxx#L61) which is the naming convention used in aod but it's not used in the reduced files where instead we add the label "Reduced".

Reply:
O2 is the suffix of all derived tables, so that should be fine. It is added automatically. How did you get "Reduced"?

Reply:
You can follow the example here:
https://github.com/AliceO2Group/O2Physics/blob/master/PWGCF/DataModel/CorrelationsDerived.h

Important is simply that the first and third parameter are the same apart from the capitalization and the "s". Note that this limits the length of your classnames to 14 characters.

Reply:
I see. Then I think the problem is when we want to save our tables we explicitly give them labels that do not follow this naming convention. Since I am still new to this I do not have all details but I will look into it and fix it in a new PR